daiquiri/output.py

```diff
@@ -41,6 +43,41 @@ def add_to_logger(self, logger):
         logger.addHandler(self.handler)
 
 
+def _get_log_file_path(logfile=None, logdir=None, program_name=None,
+                       logfile_suffix=".log"):
+    ret_path = None
+
+    if not logdir:
+        ret_path = logfile
+
+    if not ret_path and logfile and logdir:
+        ret_path = os.path.join(logdir, logfile)
+
+    if not ret_path and logdir:
+        program_name = program_name or get_program_name()
+        ret_path = os.path.join(logdir, program_name) + logfile_suffix
+
+    if not ret_path:
+        raise ValueError("Unable to determine log file destination")
+
+    return ret_path
+
+
+def _timedelta_to_seconds(td):
+    """Convert a datetime.timedelta object into a seconds interval for rotating
+    file ouput.
+
+    :param td: datetime.timedelta
+    :return: time in seconds
+    :rtype: int
+    """
+    if isinstance(td, (int, float)):
```

Comment:
You forgot `long` (Python 2). Better use `numbers.Real`, that way you sure you miss nothing. :)